src/components/Header/header.module.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+  align-items: center;
+  background-color: #f0f0f0;

```

Comment:
define color in the constant.scss and use it here

Reply:
Addressed